pages/content.vue

```diff
@@ -229,9 +269,9 @@ Sincerely,`
           if (this.$route.params.people) {
             const {people} = this.$route.params
             if (people.length === 1) {
-              mailto = people[0].email + '?'
+              mailto = people[0].email || '' + '?'
             } else if (people.length > 1) {
-              mailto = '?bcc=' + people.map(person => person.email).join(',') + '&'
+              mailto = '?bcc=' + people.map(person => person.email || '').join(',') + '&'
```

Comment:
I tested sending to multiple contacts without this change and I don't see any nulls. .map is handling that for us. So let's remove this line.